src/js/verkleFFIBindings/getTreeKey.ts

```diff
@@ -0,0 +1,115 @@
+import { Context } from '../../../dist/cjs/wasm/rust_verkle_wasm'
+
```

Comment:
This file is mainly just comments since the python version is terse but not easy to understand